Gemfile.lock

```diff
@@ -1,8 +1,3 @@
-PATH
-  remote: .
-  specs:
-    scryfall-sdk (0.0.0)
-
```

Comment:
I'm not sure what this was, but it got removed when I ran `bin/setup`, which runs `gem install -g`, so it updated the file.